### PR TITLE
fix: moving ticket up and down in wizard step 1 

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/6304
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Now, it doesn't throw an error, but I have to click it twice to move up. First it also shows down arrow, which it shouldn't because it is the last item, but once I click up, it doesn't move but down arrow disappears, and now clicking on up arrow works

File: app/components/forms/wizard/basic-details-step.js [hunk 1]
```diff
@@ -487,6 +491,7 @@ export default Component.extend(FormMixin, EventWizardMixin, {
           ticket.set('position', ticket.get('position') - 1);
         }
       });
+      this.totalTickets = this.totalTickets - 1;
```
Comment (iamareebjamal) [inline app/components/forms/wizard/basic-details-step.js:494]: This should be updated automatically

File: app/components/forms/wizard/basic-details-step.js [hunk 2]
```diff
@@ -45,6 +45,10 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     return ['Venue', 'Mixed'].includes(this.selectedLocationType);
   }),
 
+  totalTickets: computed('data.event.tickets', function() {
```
Comment (iamareebjamal) [inline app/components/forms/wizard/basic-details-step.js:48]:
```suggestion
  totalTickets: computed('data.event.tickets.[]', function() {
```

**Comment by iamareebjamal** [comment]
Still same issue

**Comment by maze-runnar** [comment]
I am trying this. it is returning correct value in js file but in hbs it's not
```
  totalTickets: computed('data.event.tickets', function() {
    return this.data.event.tickets.length;
  }),

this.deletedTickets.push(deleteTicket);
      this.totalTickets = this.totalTickets - (this.deletedTickets.length);
```

**Comment by iamareebjamal** [comment]
Ditch up down buttons, use this https://github.com/adopted-ember-addons/ember-sortable

**Comment by iamareebjamal** [comment]
We're already using https://github.com/mharris717/ember-drag-drop, so use this instead

**Comment by maze-runnar** [comment]
ok

**Comment by iamareebjamal** [comment]
Superseded by #6466

**Pull request closed without merging**